src/services/coinGecko.js

```diff
@@ -44,7 +44,7 @@ const BTC_ID = 'bitcoin';
 const WBTC_ID = 'wrapped-bitcoin';
 
 const requestConfig = {
-  timeout: 5000,
+  timeout: 10000,
```

Comment:
If we remove the timeout parameter altogether, i believe this will result in the request continuously trying... is this a better behaviour than a 10 second timeout maybe?

Reply:
Yes, if we remove it it will do it until gets dropped by native handler, but as you're saying, it would go for longer, however, downside is that we lose control here.